tsdb/head_test.go

```diff
@@ -3051,7 +3055,12 @@ func TestHistogramInWALAndMmapChunk(t *testing.T) {
 			h.NegativeBuckets = h.PositiveBuckets
 			_, err := app.AppendHistogram(0, s2, int64(ts), nil, h)
 			require.NoError(t, err)
-			exp[k2] = append(exp[k2], sample{t: int64(ts), fh: h.Copy()})
+			eh := h.Copy()
+			if !gauge && ts%30 == 1 {
```

Comment:
Same as the other comment
```suggestion
			if !gauge && ts%20 == 1 {
```

Reply:
Changed here, too.